src/Persisters/Entity/BasicEntityPersister.php

```diff
@@ -1929,8 +1930,13 @@ private function getTypes(string $field, mixed $value, ClassMetadata $class): ar
         return $types;
     }
 
-    /** @phpstan-return ArrayParameterType::* */
-    private function getArrayBindingType(ParameterType|int|string $type): ArrayParameterType|int
+    /**
+     * @psalm-return ArrayParameterType::*
+     *
+     * @throws QueryException
+     * @throws Exception
```

Comment:
In what situations is this method raising these exception? Is the caller supposed to handle it?

Reply:
Doctrine\DBAL\Types\Type::getType method throws Exception
<img width="515" alt="Screenshot 2024-12-10 at 11 54 45" src="https://github.com/user-attachments/assets/e9b0b1b7-f0c2-4ab1-828f-91d4157d5ac9">

Reply:
This answers neither of my questions. 😉

Reply:
This method doesn't handle these exceptions directly. I included them to show which ones the method can raise. If you think this is unnecessary, I’ll remove them.

Reply:
removed.